<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SETwistHelixEditorGUIClass</class>
 <widget class="QWidget" name="SETwistHelixEditorGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>125</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>125</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButtonMinusBP">
       <property name="toolTip">
        <string>Check to twist by a base pair turn in negative direction.</string>
       </property>
       <property name="text">
        <string> -1BP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButtonPlusBP">
       <property name="toolTip">
        <string>Check to twist by a base pair turn in positive direction.</string>
       </property>
       <property name="text">
        <string>+ 1BP</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioButtonTwistAngle">
       <property name="toolTip">
        <string>Check to twist by an angle.</string>
       </property>
       <property name="text">
        <string>Twist Angle</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxAngle">
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="radioButtonTwistTurns">
       <property name="toolTip">
        <string>Check to twist by the number of base pair turns.</string>
       </property>
       <property name="text">
        <string>Twist Turns</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBoxTurns">
       <property name="minimum">
        <number>-99</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>doubleSpinBoxAngle</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SETwistHelixEditorGUIClass</receiver>
   <slot>onTwistAngleChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxTurns</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SETwistHelixEditorGUIClass</receiver>
   <slot>onTwistTurnsChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonPlusBP</sender>
   <signal>toggled(bool)</signal>
   <receiver>SETwistHelixEditorGUIClass</receiver>
   <slot>onPlus(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonMinusBP</sender>
   <signal>toggled(bool)</signal>
   <receiver>SETwistHelixEditorGUIClass</receiver>
   <slot>onMinus(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonTwistAngle</sender>
   <signal>toggled(bool)</signal>
   <receiver>SETwistHelixEditorGUIClass</receiver>
   <slot>onTwistAngle(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonTwistTurns</sender>
   <signal>toggled(bool)</signal>
   <receiver>SETwistHelixEditorGUIClass</receiver>
   <slot>onTwistTurns(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onMinus(bool)</slot>
  <slot>onPlus(bool)</slot>
  <slot>onTwistAngle(bool)</slot>
  <slot>onTwistTurns(bool)</slot>
  <slot>onTwistAngleChanged(double)</slot>
  <slot>onTwistTurnsChanged(int)</slot>
 </slots>
</ui>
